src/odemis/driver/ephemeron.py

```diff
@@ -0,0 +1,788 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 24 April 2024
+
+@author: Stefan Sneep
+
+Copyright © 2024 Stefan Sneep, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the
+GNU General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis.
+If not, see http://www.gnu.org/licenses/.
+'''
+# this driver is developed for communication with an EBIC controller API
+# the EBIC scan controller from Ephemeron will acquire a digital EBIC signal
+# to be used simultaneously with other signals such as SE/CL
+
+import logging
+import os
+import queue
+import threading
+import time
+import weakref
+from typing import Any, Optional
+
+import asyncio
+from asyncua import Client, Server, Node, ua
+from asyncua.common.methods import uamethod
+from asyncua.common.statemachine import State, StateMachine, Transition
+import numpy
+from scipy import ndimage
+
+import functions
```

Comment:
That file is missing. That said, we don't want it as-is anyway. Maybe we can just copy paste the useful parts here? Otherwise, we could have a _ephemeron_functions.py file.